src/common/components/download-trigger/index.tsx

```diff
@@ -2,10 +2,31 @@ import React, {Component, Fragment} from 'react';
 
 import {Modal} from 'react-bootstrap';
 
+import BaseComponent from "../base";
+
+import {geLatestDesktopTag} from "../../api/misc";
+
 import platform from '../../util/platform';
 
-export class DialogContent extends Component {
+interface ContentState {
+    desktopTag: string;
+}
+
+export class DialogContent extends BaseComponent<{}, ContentState> {
+
+    state: ContentState = {
+        desktopTag: "3.0.14"
```

Comment:
3.0.15 should be default because of api path change. if github api had issues, user will get old not working version.

Reply:
done